examples/inverse/plot_custom_inverse_solver.py

```diff
@@ -40,8 +40,8 @@
 
 evoked = evoked.pick_types(eeg=False, meg=True)
 # Handling forward solution
-forward = mne.read_forward_solution(fwd_fname, surf_ori=True)
-
+forward = mne.read_forward_solution(fwd_fname)
+forward = mne.convert_forward_solution(forward, surf_ori=True)
```

Comment:
Ahh I see. No, just please make sure that any tests and examples that throw warnings or will change behavior when we change the default have their value set explicitly.